packages/commercetools/api-client/src/types/setup.ts

```diff
@@ -53,8 +58,10 @@ export interface SetupConfig<TCacheShape> {
   locales?: LocaleItem[];
   languageMap?: Record<string, any>;
   acceptLanguage?: string[];
+  store: string;
   cookies?: CookiesConfig;
   auth?: Auth;
+  stores?: Stores;
```

Comment:
It sounds a bit like `Array<Store>` for me. Maybe it would be worth refactoring this name to make sure developers won't be confused? Wdyt?